example/monster/run-hot-import-bot.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
Please change the script name from `run-hot-import-bot` to `run-monster-bot` for making it more clear for this example directory.